measureme/src/file_header.rs

```diff
@@ -6,7 +6,7 @@ use crate::serialization::SerializationSink;
 use byteorder::{ByteOrder, LittleEndian};
 use std::error::Error;
 
-pub const CURRENT_FILE_FORMAT_VERSION: u32 = 1;
+pub const CURRENT_FILE_FORMAT_VERSION: u32 = 2;
```

Comment:
I don't think we have adopted a formal policy yet, but since we haven't yet released format `1`, does this actually need to be incremented?

Reply:
I think the policy should be that every PR keeps this consistent. It's not much work and there is no harm in jumping a few version numbers in between releases.